### DISCUSSION TITLE
Collaberate on Image-to-Image Generation

### DISCUSSION THREAD

**Discussion by lawrence-cj** [PixArt-alpha/PixArt-alpha#47] [Ideas]
We encourage anyone interested in PixArt-alpha to help us design a PixArt-Controlnet or PixArt-I2I model. All viable programs are welcome. Appreciated!

**Comment by ArshCypherZ** [comment]
sure, you can get help from ai like [enhanceai](https://enhanceai.art), stable diffusion and much moree
